WORKSPACE

```diff
@@ -122,25 +118,27 @@ load("//dependencies/vaticle:artifacts.bzl", "vaticle_typedb_artifact", "vaticle
 vaticle_typedb_artifact()
 vaticle_typedb_cluster_artifact()
 
-# Load maven
-load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_artifacts = "artifacts")
-load("@vaticle_typeql//dependencies/maven:artifacts.bzl", vaticle_typeql_artifacts = "artifacts")
-load("@vaticle_factory_tracing//dependencies/maven:artifacts.bzl", vaticle_factory_tracing_artifacts = "artifacts")
-load("//dependencies/maven:artifacts.bzl", vaticle_typedb_client_java_artifacts = "artifacts", vaticle_typedb_client_java_overrides = "overrides")
-
 ###############
 # Load @maven #
 ###############
```

Comment:
Group together all maven related statements in WORKSPACE file. All statements under this comment are related to maven. There are no maven related statements outside of this section.